redream

```diff
@@ -166,6 +191,18 @@ if [[ "$PLATFORM" == "mac" ]]; then
 	create_macos_application
 elif [[ "$PLATFORM" == "linux" ]]; then
 	create_linux_desktop_launcher
+	if [[ "$REDREAM_ARCH" == "universal-raspberry" ]]; then
+		if [[ $launch_args == *"system-mesa"* ]]; then
+			#exec "$REDREAM_DIR/redream"
+			echo "Using the system library over the bundled one..."
+			cp $REDREAM_DIR/releases/v*/redream.$REDREAM_ARM_TYPE.elf $REDREAM_DIR/redream
```

Comment:
To copy the correct file from the extracted release you can use `$RELEASE_PATH`. Quoting the paths will also ensure compatibility for installations where either path contains spaces, e.g.

```sh
cp "$RELEASE_PATH/redream.$REDREAM_ARM_TYPE.elf" "$REDREAM_DIR/redream"
```